File: ConferenciasApi/Models/Registro.cs
namespace ConferenciasApi.Models;

public class Registro
{
    public int RegistroId { get; set; }

    public int ConferenciaId { get; set; }
    public Conferencia? Conferencia { get; set; }

    public int AsistenteId { get; set; }
    public Asistente? Asistente { get; set; }

    public DateTime FechaRegistro { get; set; } = DateTime.Now;
}

File: conferencias-front/src/app/models/registro.ts
export interface Registro {
  registroId: number;
  fechaRegistro: string;
  conferencia: {
    conferenciaId: number;
    nombre: string;
    fecha: string;
    ubicacion: string;
  };
  asistente: {
    asistenteId: number;
    nombre: string;
    apellido: string;
    email: string;
  };
}